### ISSUE TITLE
Rushil and Priyansh - Model Testing

### ISSUE THREAD

**Issue by margoglvz** [CareTech-at-UCI/melanoma-project#2]
# Instructions for Task 4 - Model Testing and Calculations

## **Overview:**
Congrats on getting your model training done! Next step is to test with the ISIC 2019 testing dataset and evaluate metrics to see your final model prediction results.

### **Deadline:**
The deadline to finish the model testing is by **next Friday, 11:59PM.** It is important to try and finish everything related with your model by next Friday so that it is a smoother process to start the web development tasks.

*****If you are still working on your model training task**, please make sure to finish ASAP! This is so the model testing is not as stressful and we can move at the same pace

### **Instructions:**
- **Git Branch**
    - For the testing phase, you can work on the same branch. Once the model is done then I will work on merging.
- **Working with Testing Dataset**
  For the final testing phase, you will have to:
   - Download the ISIC 2019 Testing Dataset
   - Preprocess all the photos
   - Run the preprocessed photos to do final testing
   - Calculate F1 Score, Accuracy, and Confusion Matrix
- **How to Calculate F1 Score, Accuracy, and Confusion Matrix**
   - SKLEARN 😀
   - `from sklearn.metrics import confusion_matrix, accuracy_score, f1_score`
   - Use sklearn.metrics to find out these metrics for your model to have a final evaluation of your model.
   - **IDEA:** In order to have everyone’s model included, I was thinking of making it so the user can choose which model they can choose! So they 
  can basically get different opinions. This is not final but I will be working on that more later 👍
   - **_Make sure to clearly print these somewhere in your code for me to see so I can do a final evaluation on everyone’s models, thank you!_**
   
- **Links for downloading (~12GB Total):**
  https://challenge.isic-archive.com/data/#2019

Thank you for reading and goodluck!
